spec/features/create_and_update_workflow_spec.rb

```diff
@@ -10,9 +10,11 @@
     find("a", text: "Create a new workflow").click
     within("#add-workflow-modal") do
       fill_in 'Name', with: 'New workflow'
-      # find('#hasComment', visible: :all).trigger('click')
-      # find('#reportable', visible: :all).trigger('click')
-      # find('#multi_team_quant_essential', visible: :all).trigger('click')
+      find('#active', visible: :all).first(:xpath, './/..').click
+      find('#reportable', visible: :all).first(:xpath, './/..').click
+      find('#hasComment', visible: :all).first(:xpath, './/..').click
+      find('#qcFlow', visible: :all).first(:xpath, './/..').click
+      find('#cherrypickFlow', visible: :all).first(:xpath, './/..').click
       find("button", text: "Create").click
```

Comment:
Could test the initial state name functionality here.

Reply:
Yes.

Reply:
Done in 452c548